spec/mock/middleware/validate-request-body-middleware-mock.js

```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+var container;
+
+container = require("../../../lib/dependencies");
+
+module.exports = () => {
+    var middleware, middlewareFactory;
+
+    middleware = jasmine.createSpy("validate-request-body-middleware-mock-middleware").andCallFake((req, res, next) => {
```

Comment:
Other places you create a spy you do camel casing, we should match here as well.